packages/truffle-debugger/lib/session/reducers.js

```diff
@@ -11,24 +11,45 @@ import * as actions from "./actions";
 export const WAITING = "WAITING";
 export const ACTIVE = "ACTIVE";
 export const ERROR = "ERROR";
-export const FINISHED = "FINISHED";
 
-export function session(state = WAITING, action) {
+export function status(state = WAITING, action) {
+  //theState to avoid name collision
```

Comment:
what's this comment about?

Reply:
Oops.  Outdated comment from intermediate version, looks like.  Will fix.

Reply:
(Originally I had it as `session.state`, but that left the problem of what to call the variable `state`, so I renamed it to `theState`.  Eventually I realized that `status` was a suitable synonym for `state, avoiding the collision problem.)